tsl/test/expected/cagg_plugin.out

```diff
@@ -0,0 +1,308 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+\c :TEST_DBNAME :ROLE_SUPERUSER
+-- Test the invalidation plugin and make sure it can return reasonable
+-- data. We do that by creating a hypertable and then registering a
```

Comment:
```suggestion
-- Test the invalidation plugin and make sure it can return the
-- data we expect. We do that by creating a hypertable and then registering a
```